.projenrc.ts

```diff
@@ -18,7 +18,8 @@ const repo = new YarnMonorepo({
       trailingComma: pj.javascript.TrailingComma.ALL,
     },
   },
-  workflowRunsOn: ['awscdk-service-spec_ubuntu-latest_32-core'],
+  // These runners are currently not working, so let's disable them for now.
+  // workflowRunsOn: ['awscdk-service-spec_ubuntu-latest_32-core'],
```

Comment:
Can we try this? I might pick the one that's available...

```suggestion
  workflowRunsOn: ['awscdk-service-spec_ubuntu-latest_32-core', 'ubuntu-latest'],
```